src/tools/rbd/Schedule.cc

```diff
@@ -230,23 +217,17 @@ int Schedule::parse(json_spirit::mValue &schedule_val) {
 }
 
 void Schedule::dump(ceph::Formatter *f) {
-  f->open_array_section("items");
-  for (auto &item : items) {
-    f->open_object_section("item");
-    f->dump_string("interval", item.first);
-    f->dump_string("start_time", item.second);
-    f->close_section(); // item
+  f->open_array_section("intervals");
```

Comment:
Do you think it's worth highlighting the change in the CLI structured output (`items` -> `intervals`, flat structure) in the release note?